packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -417,7 +419,7 @@ class TimelineFrame {
 
   int get cpuDuration => _cpuEventFlow?.duration;
 
-  double get cpuDurationMs => cpuDuration / 1000;
+  double get cpuDurationMs => cpuDuration != null ? cpuDuration / 1000 : null;
```

Comment:
add a TODO to simplify the API on this class. I think we can reduce the amount of duplicated logic for CPU and gpu but it doesn't need to happen now.

Reply:
done.